content/terraform/v1.12.x/docs/language/block/stack/deploy/deployment.mdx

```diff
@@ -0,0 +1,294 @@
+---
+page_title: deployment block reference for Terraform Stack deployments
+description: Use the `deployment` block to define how many times you want to deploy your Stack's infrastructure.
+---
+
+# `deployment` block reference
+
+Use the `deployment` block to define how many times you want to deploy your Stack's infrastructure. You can define `deployment` blocks in a `<NAME>.tfdeploy.hcl` file, which is the deployment configuration file for your Stack.
+
+## Background
+
+<Note>
+
+HCP Terraform supports up to a maximum of 20 deployments per Stack.
+
+</Note>
+
+The `deployment` block defines how to deploy your Stack's infrastructure. Each deployment represents a separate instance of your Stack infrastructure. Each deployment runs in its own isolated agent, wholly separated from other Stack deployments.
```

Comment:
Hmm, "isolated agent" isn't quite right. I mean, every plan that runs has some isolation, but we don't lock a deployment to a specific agent. 

```suggestion
The `deployment` block defines how to deploy your Stack's infrastructure. Each deployment represents a separate instance of your Stack infrastructure. Each deployment maintains its own isolated state data, wholly separated from other Stack deployments.
```

I think that's probably closer to what's meant?